cmd/notation/blob/inspect_test.go

```diff
@@ -0,0 +1,56 @@
+package blob
+
+import (
+	"github.com/notaryproject/notation/internal/cmd"
+	"reflect"
+	"testing"
+)
+
+func TestBlobInspectCommand_SecretsFromArgs(t *testing.T) {
```

Comment:
We dont have any secret in inspect command.

Reply:
Removed in updated rev